lib/Doctrine/Expression/Pgsql.php

```diff
@@ -73,7 +73,6 @@ public function substring($value, $from, $len = null)
         $value = $this->getIdentifier($value);
 
         if ($len === null) {
-            $len = $this->getIdentifier($len);
```

Comment:
This was commented in phpstan ignores, but this case doesn't make much sense. `getIdentifier` doesn't accept `null` (in this case it returns `null`), and then `$len` is never used after it's set.